RHEL6_7/system/scl-collection/module.ini

```diff
@@ -1,7 +1,7 @@
 [preupgrade]
 content_title: Red Hat Software Collections
 author: Petr Hracek <phracek@redhat.com>
-content_description: The module checks if RHSCL are installed.
+content_description: The module checks if RHSCL is installed.
```

Comment:
RHSCL is nothing specific. It's more like a named group of packages, and there are many of these _groups_ available for installation. So I would probably add `any` -> `The module checks if any RHSCL is installed.`